allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
You solved the issue, but in the wrong way. It will be harder to support future changes.

I even have an alternative for you. Increase line-height for `.table__col` which has the same effect.